src/context/TranscriberContext.tsx

```diff
@@ -504,7 +504,7 @@ const TranscriberProvider = withData(mapRecordsToProps)(
     useEffect(() => {
       const playItem = state.playItem;
       const rowList: IRowData[] = [];
-      if (pasId && artifactId) {
+      if (pasId) {
```

Comment:
Did you try this with the original transcriber?  I think there you have a pasId as the current one, but don't want to limit your tasks to the current one...but I could be wrong and this might work.

Reply:
I added isDetail